### DISCUSSION TITLE
What could cause a huge CPU and disk usage spike over several hours? (self-hosting newb)

### DISCUSSION THREAD

**Discussion by spencerflagg** [getfider/fider#1063] [Q&A]
My Fider instance, running alone in Docker on a VPS, just experienced a huge spike in CPU and disk usage over something like 8 to 12 hours. During this time, Fider's app container went to an Unhealthy state, and became unreachable in the browser.

After a restart of both the app container and the db container, everything is back to normal.

**Wondering what could cause this, and more importantly, how to debug and find relevant logs on my own in the future.**

Thanks in advance!

_Screenshot of the hardware activity - doesn't appear to be associated with network activity, which surprises me._
![image](https://user-images.githubusercontent.com/5099192/159191818-099cd7d3-fd32-4c4c-92a9-58cb765167ec.png)

**Comment by goenning** [comment]
Next time it happens, check how many open transactions are in the DB and what state they are.

I’ve seen complaints about it but I never experienced something like that myself
